zp-relayer/utils/helpers.ts

```diff
@@ -248,3 +248,9 @@ export function getFileHash(path: string) {
   hash.update(buffer)
   return hash.digest('hex')
 }
+
+export function applyDenominator(n: BN, d: BN) {
+  return d.isNeg()
+    ? n.div(d.neg())
+    : n.mul(d)
+}
```

Comment:
Maybe this way is shorter?
```suggestion
export function applyDenominator(n: BN, d: BN) {
  return d.testn(255)
    ? n.div(d.maskn(255))
    : n.mul(d)
}
```

Reply:
But we also need to do a reverse of `applyDenominator` here, in `FeeManager`, which is easier to do with simple `.neg()` call
https://github.com/zkBob/zeropool-relayer/blob/2ae48d9e7e4a539ce59b3f56a4cb42b5989d696e/zp-relayer/services/fee/FeeManager.ts#L48

Reply:
I mean, that BN.js doesn't store the sign in 255th bit, as far as I know

Reply:
But, I guess we still can do a xor here, in `FeeManager`, instead of `.neg()`

Reply:
I refactored it a bit, as you suggested